packages/bits/demo/src/components/app/app-routing.module.ts

```diff
@@ -31,357 +31,399 @@ const appRoutes: Routes = [
         loadChildren: async () =>
             import("../../../src/static/static-example.module").then(
```

Comment:
should ng15 allow ommiting the then part when you make the ng module the default export?

Reply:
yes you are right it is available  
it will be much cleaner 
        loadChildren: async () =>
            import("../../../src/static/static-example.module")
            
I will clean it up